Mathlib/RingTheory/WittVector/IsPoly.lean

```diff
@@ -328,8 +328,7 @@ theorem IsPoly.map [Fact p.Prime] {f} (hf : IsPoly p f) (g : R →+* S) (x : 
   ext n
   simp only [map_coeff, hf, map_aeval]
   apply eval₂Hom_congr (RingHom.ext_int _ _) _ rfl
-  ext  -- Porting note: this `ext` was not present in the mathport output
-  simp only [map_coeff]
+  rfl
```

Comment:
I guess this is fine, but please double-check.

Reply:
Either `rfl` or keep `ext` and delete the porting note entirely: looks like it only used to work because the goal looked something like `(fun i ↦ g (x.coeff i)) = (fun i ↦ ((WittVector.map g) x).coeff i)`, and now the RHS gets eta-reduced into `((WittVector.map g) x).coeff` at some point.

Reply:
`rfl` then